examples/feature_alignment_example/misalign_data.py

```diff
@@ -0,0 +1,40 @@
+import random
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+
+
+def random_split_data(df: pd.DataFrame, n: int) -> List[pd.DataFrame]:
+    df_rand = df.sample(frac=1.0, random_state=42)
+    num_rows_per_df = len(df_rand) // n
+    smaller_dfs = [df_rand.iloc[i * num_rows_per_df : (i + 1) * num_rows_per_df] for i in range(n - 1)]
+    smaller_dfs.append(df_rand.iloc[(n - 1) * num_rows_per_df :])
+    return smaller_dfs
+
+
+if __name__ == "__main__":
+    data_path = Path("examples/feature_alignment_example/mimic3d.csv")
+    target_datapath = "examples/feature_alignment_example"
+
+    df = pd.read_csv(data_path)
+    df.dropna(inplace=True, how="any")
+    dfs = random_split_data(df, 2)
+    df1 = dfs[0].reset_index()
+    df2 = dfs[1].reset_index()
+
+    df2 = df2.drop(columns=["ExpiredHospital", "admit_type", "NumRx", "ethnicity"])
+
+    # Now we randomly select 10 percent of the rows of df2
+    # and set its 'insurance' column to 'Unknown'
+    num_rows_to_select = int(0.10 * len(df2))
+    random_indices = random.sample(range(len(df2)), num_rows_to_select)
+
+    # Set a specific column ('B' in this example) to the specific value for the selected rows
+    df2.loc[random_indices, "insurance"] = "Unknown"
+
+    print(f"hospital1 insurance values: {df1['insurance'].unique()}")
```

Comment:
If you think it's useful to have this in the script, let's use `log(INFO, f"...")` instead of print